app/services/recording_creator.rb

```diff
@@ -68,7 +68,7 @@ def get_recording_length(recording:)
     length = 0
     if recording[:playback][:format].is_a?(Array)
       recording[:playback][:format].each do |formats|
-        length = formats[:length] if formats[:type] == 'presentation'
+        length = formats[:length] if formats[:type] == 'presentation' || formats[:type] == 'video'
```

Comment:
Is it necessary to use specific formats here? Besides presentation or video it could also be podcast, notes, screenshare or potentially a custom format.

Reply:
Actually, nevermind. The other formats probably don't provide a length value? I'm also wondering what is it even used for?